examples/transpose/transpose_serial.cpp

```diff
@@ -152,8 +152,8 @@ int main(int argc, char* argv[])
         ( "verbose", "Verbose output")
     ;
 
-    // Initialize and run HPX, this example is serial and therefor only needs on thread,
-    // We just use hpx::init to parse our command line arguments
+    // Initialize and run HPX, this example is serial and therefore only needs
+    // on thread, We just use hpx::init to parse our command line arguments
```

Comment:
Should it be `one thread` instead?

Reply:
It helps to read around the typo as well... thanks!